velox/docs/functions/spark/datetime.rst

```diff
@@ -49,6 +49,14 @@ These functions support TIMESTAMP and DATE input types.
     ``day`` need to be from 1 to 31, and matches the number of days in each month.
     days of ``year-month-day - 1970-01-01`` need to be in the range of INTEGER type.
 
+.. spark:function:: next_day(start_date, dayofweek) -> date
+   
+    Returns the first date which is later than ``start_date`` and named as indicated.
+    Returns NULL if day_of_week is an invalid input. ::
```

Comment:
What if start_date is an invalid input?

Reply:
It can't be invalid, start_date is Date, which is int32, not string

Reply:
It can be -2

Reply:
-2 is also validate